platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/PcesFile.java

```diff
@@ -371,14 +453,15 @@ public void deleteFile(@NonNull final Path rootDirectory, @Nullable final Recycl
 
     /**
      * Get an iterator that walks over the events in this file. The iterator will only return events that have a
-     * generation equal to or greater to the minimum generation.
+     * are not sequentially preceding the lower bound.
```

Comment:
The iterator will only return events that have a are not sequentially preceding the lower bound."

grammar is off

Reply:
fixed